cedar-policy-core/src/ast/entity.rs

```diff
@@ -588,36 +608,29 @@ mod test {
     fn test_euid_equality() {
         let e1 = EntityUID::with_eid("foo");
         let e2 = EntityUID::from_components(
-            Name::parse_unqualified_name("test_entity_type").expect("should be a valid identifier"),
+            Name::parse_unqualified_name("test_entity_type")
+                .expect("should be a valid identifier")
+                .into(),
             Eid("foo".into()),
             None,
         );
-        let e3 = EntityUID::unspecified_from_eid(Eid("foo".into()));
-        let e4 = EntityUID::unspecified_from_eid(Eid("bar".into()));
```

Comment:
Nit: it bothers me that we have `e1`, `e2`, and `e5` now 😂 This test was primarily testing unspecified entities, so you can probably just delete it entirely.